xbmc/pvr/addons/PVRClient.h

```diff
@@ -165,7 +165,7 @@ class CPVRClient : public ADDON::IAddonInstanceHandler
   /*!
    * @return A friendly name for this add-on that can be used in log messages.
```

Comment:
Mind changing the comment to reflect that the friendly name is also usable for GUI purposes?

Reply:
Sure that reminds me the sometimes ID() is used for log messages and it doesn't show the instance but that is another PR